16_wordle/main_test.go

```diff
@@ -35,3 +37,33 @@ func TestContains(t *testing.T) {
 		}
 	}
 }
+
+func TestNewList(t *testing.T) {
+	// tmpDir := t.TempDir()
+	err := os.WriteFile("t.TempDir(dir/list)", []byte("brown"), []byte("rossa"), 0666)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	got, _ := NewList("dir/list")
+
+	if got != List{
+				words: [][]byte{
+					[]byte("brown", []byte("rossa")),
+		{
+			t.Errorf("NewList = %v", got)
```

Comment:
Make a new variable for your [][]byte list of words called "want", and use it everywhere it's needed.

```suggestion
			t.Errorf("want: %v but got: %v", got)
```

Reply:
Done